api/indexer.oas2.json

```diff
@@ -679,7 +679,7 @@
     },
     "/v2/assets/{asset-id}/transactions": {
       "get": {
-        "description": "Lookup transactions for an asset.",
+        "description": "Lookup transactions for an asset. Transactions are returned oldest to newest.",
```

Comment:
is there a reason the order is inconsistent with /v2/accounts/{account-id}/transactions?

Reply:
Yes, basically, the txn/account index is optimized the "wallet" scenario and returns things newest to oldest. All other queries are done oldest to newest. The README goes into a little more detail about it.